pkg/util/k8s/node.go

```diff
@@ -65,14 +82,17 @@ func IsPodRecentlyCreatedAfterNodeCordoned(
 		return true
 	}
 
-	lastCreationTime, ok := lastCreationTimeOnNode[node.Name]
-	// If the pod has never been created, no timestamp.
-	if !ok {
-		return false
+	// If pod has been recently created, return true
+	// Otherwise the pod will get deleted, increase the restart delay
```

Comment:
The pod won't get deleted, it will return shouldRun false, shouldContinueRun true by caller. 
"Otherwise the pod will get deleted"

Reply:
it has different callers, will add comments at the caller.